central/image/service/service_impl.go

```diff
@@ -455,6 +456,9 @@ func (s *serviceImpl) EnrichLocalImageInternal(ctx context.Context, request *v1.
 				s.informScanWaiter(request.GetRequestId(), nil, err)
 				return nil, err
 			}
+		} else {
+			// If we didn't update the scan, fill in the stats from existing image (if there is one).
+			enricher.FillScanStats(img)
```

Comment:
same here

Reply:
Same comment as above, believe it was a bug.  With out this the component count on the image will always be zero.